compiler/src/iree/compiler/Codegen/LLVMCPU/test/apply_scale_lowering.mlir

```diff
@@ -0,0 +1,187 @@
+// RUN: iree-opt --split-input-file --pass-pipeline='hal.executable(hal.executable.variant(builtin.module(iree-convert-to-llvm)))' %s | FileCheck %s
+
+#executable_layout = #hal.executable.layout<push_constants = 0, sets = [
+  #hal.descriptor_set.layout<0, bindings = [
+    #hal.descriptor_set.binding<0, storage_buffer>,
+    #hal.descriptor_set.binding<1, storage_buffer>
+  ]>
+]>
+#executable_target_embedded_elf_riscv_64_ = #hal.executable.target<"llvm", "embedded-elf-riscv_64", {
+  cpu_features = "+m,+a,+f,+d,+c",
+  data_layout = "e-m:e-p:64:64-i64:64-i128:128-n64-S128",
+  native_vector_size = 256 : index,
+  target_triple = "riscv64-unknown-unknown-eabi-elf"
+}>
+#map = affine_map<()[s0] -> (s0 ceildiv 2)>
+#translation = #iree_codegen.translation_info<CPUDoubleTilingExpert workload_per_wg = [2]>
+hal.executable private @apply_scale_no_vector_feature {
+  hal.executable.variant public @embedded_elf_riscv_64, target = #executable_target_embedded_elf_riscv_64_ {
+    hal.executable.export public @apply_scale_no_vector_feature ordinal(0) layout(#executable_layout) {translation_info = #translation} {
+    ^bb0(%arg0: !hal.device, %arg1: index, %arg2: index, %arg3: index):
+      %c1 = arith.constant 1 : index
+      %0 = affine.apply #map()[%arg1]
+      hal.return %0, %c1, %c1 : index, index, index
+    }
+    builtin.module {
+      func.func @apply_scale_no_vector_feature() {
+        %cst = arith.constant dense<19689> : vector<2xi32>
+        %cst_0 = arith.constant dense<15> : vector<2xi8>
+        %c0 = arith.constant 0 : index
+        %0 = hal.interface.binding.subspan set(0) binding(0) type(storage_buffer) offset(%c0) alignment(64) : memref<2xi32>
+        %1 = hal.interface.binding.subspan set(0) binding(1) type(storage_buffer) offset(%c0) alignment(64) : memref<2xi32>
+        %2 = vector.load %0[%c0] : memref<2xi32>, vector<2xi32>
+        %3 = "tosa.apply_scale"(%2, %cst, %cst_0) {double_round = false} : (vector<2xi32>, vector<2xi32>, vector<2xi8>) -> vector<2xi32>
+        vector.store %3, %1[%c0] : memref<2xi32>, vector<2xi32>
+        return
+      }
+    }
+  }
+}
+
+// 64-bit lowering is used by default if no vector features are provided.
+// TODO: We shouldn't vectorize the code if no vector features are provided.
```

Comment:
nit: TODO(handle) or TODO(issue number)

Do we have an issue for this feature improvement?

Reply:
`TODO:` is LLVM style. The IREE style is using `TODO(handle)` or `TODO(#ISSUE_NUMBER)`.

Reply:
Fixed the TODO, thanks! No, we don't have an issue yet. Feel free to open one with the specific information of your use case so that we can discuss it in more detail.